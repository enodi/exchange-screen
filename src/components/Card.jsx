import React from "react";

const Card = props => <div className="card-container">{props.children}</div>;

export default Card;
