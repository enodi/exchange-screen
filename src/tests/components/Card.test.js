import React from "react";
import { shallow } from "enzyme";
import Card from "components/Card";

describe("Card Component", () => {
  let wrapper;
  const children = <h1>Hello World</h1>;
  beforeEach(() => {
    wrapper = shallow(<Card>{children}</Card>);
  });

  it("should render Card component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("renders the Card children", () => {
    expect(wrapper.find("h1").text()).toEqual("Hello World");
  });
});
