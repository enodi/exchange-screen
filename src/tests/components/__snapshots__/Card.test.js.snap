// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component should render Card component correctly 1`] = `
<div
  className="card-container"
>
  <h1>
    Hello World
  </h1>
</div>
`;
